app/src/common/shared/org/mozilla/vrbrowser/ui/widgets/Windows.java

```diff
@@ -1342,10 +1326,7 @@ public void onTabsReceived(@NonNull List<TabData> aTabs) {
 
             if (i == 0 && !fullscreen) {
                 // Set the first received tab of the list the current one.
-                SessionStore.get().setActiveSession(session);
-                targetWindow.setupListeners(session);
-                targetWindow.getSession().setActive(false);
-                targetWindow.setSession(session);
+                targetWindow.setSession(session, WindowWidget.LEAVE_CURRENT_SESSION_ACTIVE);
```

Comment:
The removed code seems to be `DEACTIVATE_CURRENT_SESSION` and not `LEAVE_CURRENT_SESSION_ACTIVE`. Am I reading it wrong?

Reply:
Right, I've updated.